kuibit/grid_data.py

```diff
@@ -2142,7 +2142,6 @@ def is_integer(x):
         # Next, we loop over all the components and check if their dx is an
         # integral multiple of dx_finest
         for comp in self.all_components:
-
```

Comment:
I think that this change is not needed. Could you please force-push and remove it?